addons/api/XEH_PREP.hpp

```diff
@@ -1,2 +1,5 @@
 // Helpers
 PREP(mapChannelFieldName);
+
+// Obsolete
+OBSOLETE_SYS(FUNC(setItemRadioReplacement),{params ['_radioType']; EGVAR(sys_radio,defaultItemRadioType) = _radioType;});
```

Comment:
Should use `CBA_settings_fnc_set`, otherwise it will get reverted saving any other settings.